sway/commands/keep_keyboard_layout.c

```diff
@@ -0,0 +1,25 @@
+#define _XOPEN_SOURCE 700
```

Comment:
This is probably not needed.

Reply:
Oh, wait a second, I this probably didn't get into the diff because I had old file with "kbd" naming opened. Let me redo this in a sec.

Reply:
Thanks, done